NuGet.config

```diff
@@ -5,6 +5,7 @@
     <add key="dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCoreDev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
```

Comment:
This shouldn't be necessary. These packages are just a later version of aspnetcore-ci-dev.